free-1.md

```diff
@@ -10,34 +11,57 @@ _What need is met by your idea? Who are you helping? What is that person's
+
+Additionally, to just the average user, this could be a great learning tool. If you want to learn how strong a certain hero is against another hero, you could use this to run damage calculations. You could also use this to test different builds (like, if I get X item before Y item, I'll farm faster but be easier to kill). As of now, the way to do this is literally just to play the game. You test different item builds, and try them in the game, most of the time leading to your death as you learn what things work and don't work. This could simoplify that process. 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
-
+As described above, the need is essentially to provide an interface that lets people test ideas in DOTA 2 quickly and easily. A GUI could work for this, but would effectively just be a GUI on top of a language that's doing all the backend computations. Thus, a first step is to make a DSL for it. Additionally, the language does not need to be turing complete, it can focus entirely on its domain. It might have batch testing and things that make use of more elaborate control flow, but these should all be in the backend. The goal is to make the interface as simple to use as possible. 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I'm excited about this first and foremost because I play the game, and think it would be awesome to have something like this. I also know lots of people who play the game as well, and would love this. Second, it would be very helpful with respect to the constant debates about things in the game that are always hard to fact-check due to the difficulty of testing these things. I came up with it just by thinking of an interesting domain, and immediately thought of DOTA 2. 
 
 ### Domain
 _Describe the project's domain in five words._
 
+DOTA Hero/Item Balance Testing
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would interact with this language just by writing scripts in either the shell, or a text editor. I'm imagining commands like:
+Hero {item1, item2, item3, item4, item5, item6} level 15;
+printing out the relevant stats for that hero. The core issue seems to be text or GUI as the core interaction. I claim that text is probably faster, as you could much mroe easily copy and paste the same 6 items into different heroes, rather than having to reselect everything every time. Text makes things like standard batch testing much easier than a GUI on its own.
 
```

Comment:
It'd be cool if you could specify at which level each item was acquired.  Then in the output you could have the hero's stats over some sort of time (or level) axis.  Then you'd be able to guage that specific build's strenghts + weaknesses in the early, mid, and late game.